cross_compile/Dockerfile_ros

```diff
@@ -72,35 +81,48 @@ RUN python3 -m pip install -U \
     pytest-cov \
     pytest-runner \
     setuptools \
-    vcstool
+    vcstool; \
+  fi
 
-# Install Fast-RTPS dependencies
-RUN apt-get update && apt-get install --no-install-recommends -y \
+# Install Fast-RTPS dependencies for ROS 2
+RUN if [[ "${ROS_VERSION}" == "ROS2" ]]; then \
+    apt-get update && apt-get install --no-install-recommends -y \
         libasio-dev \
         libtinyxml2-dev \
-    && rm -rf /var/lib/apt/lists/*
+    && rm -rf /var/lib/apt/lists/*; \
```

Comment:
I don't think you need the semicolon since there's a newline here

Reply:
Bash needs every statement to terminate in a semicolon, causes an error otherwise.